tutorial/advanced/t15_finetuning.ipynb

```diff
@@ -663,7 +654,11 @@
     "        x = fn.relu(self.backbone_layers[2](x))\n",
     "        return x\n",
     "\n",
-    "backbone_torch = fe.build(model_fn=lambda: BackboneTorch(model_torch_pretrained), optimizer_fn=\"adam\") "
```

Comment:
presumably this BackBoneTorch method should have been deleted from somewhere right?

Reply:
oh, nvm